src/SourceGenerators/Uno.UI.SourceGenerators/ApiInformation.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Windows.Foundation.Metadata
+{
+	public partial class ApiInformation
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826539)